README.md

```diff
@@ -150,19 +152,19 @@ When all the tests complete (`status` informs there are no queued commands and a
 collate apm-server.tracing.enabled branch=trace-apm-server
 ```
 
-That will show 2 groups of reports, one with `numAgents=1` and another one with `numAgents=20`. 
+That will show 2 groups of reports, one with `agents=1` and another one with `agents=20`. 
 
-If you run tests with different parameter values (events, duration, etc), you will get 1 group per variant.
+If you run tests with different parameter values (transactions, duration, etc), you will get 1 group per variant.
 Then, per each group you will see 2 reports, one with tracing enabled and other with tracing disabled:
 
 ```
-duration 30s  events 10  spans 10  frames 10  numAgents 20  branch trace-apm-server  revision c961d664fbe5893d523bd9077be64617f57a96e7
+duration 30s  transactions 10  spans 10  frames 10  agents 20  branch trace-apm-server  revision c961d664fbe5893d523bd9077be64617f57a96e7
```

Comment:
this and L167 are not exact like that, `errors` is missing. so:

`duration 30s  errors 0 transactions 10  spans 10  frames 10  agents 20  branch trace-apm-server`